citadel/ros_integration.md

```diff
@@ -74,9 +81,11 @@ setup.bash (not needed for binary packages):
 
 ```bash
 # Shell D:
-. ~/bridge_ws/install/setup.bash
+. ~/workspace/install/setup.bash
```

Comment:
earlier in this tutorial, it has a link to ros_ign's [install section](https://github.com/osrf/ros_ign/tree/melodic/ros_ign_bridge#building-the-bridge-from-source) (under Source install). In those instructions, the workspace is named `bridge_ws` so I believe the intention here is to be consistent with the name in the ros_ign's installation instructions.